app/views/schools/edit.html.erb

```diff
@@ -13,6 +13,6 @@
   <label>Zipcode: </label>
   <input type='text' name='school[zipcode]' value="<%= @school.zipcode %>"/><br/>
   <label>Grade School?: </label>
-  <input type='checkbox' name='school[gradeschool]' value="<%= @school.gradeschool %>"/><br/>
+  <input type='checkbox' name='school[gradeschool]' <%= @school.gradeschool ? 'checked' : '' %>/><br/>
```

Comment:
Nice job with this! What does the `?` do in this line? Also maybe we should indent the input lines in our HTML so it would be easier to read.